task_12/index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="theme-color" content="#e65c00">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link rel="stylesheet" href="style/css/main.css">
+    <link rel="icon" type="image/vnd.microsoft.icon" href="img/favicon.ico">
+    <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+    <title>Pizza App</title>
+</head>
+<body>
+    <header>
+        <div class="time">
+            <span>
+                <i class="far fa-clock"></i>
+                <span id="time">00:00:00</span>
+            </span>
+        </div>
+        <div class="logo">
+            <a href="#">
+                <img src="img/pizza_logo.png" alt="pizza logo">
+            </a>
+        </div>
+        <div class="login">
+            <i class="fas fa-user"></i>
+            <a href="#">login/signup</a>
+        </div>
+    </header>
+
+    <main>
+        <div class="buttons">
+            <button class="button button--stripe" type="button">
+                <i class="fas fa-plus"></i>
+                Add new pizza
+            </button>
+        </div>
+
+        <div class="wrapper">
+            <div class="pizza">
+                <img class="absolute" src="img/rucola.png" alt="rucola image">
+                <div class="pizza__image">
+                    <img src="img/1.png" alt="pizza 1 image">
+                </div>
+                <div class="pizza__text">
+                    <div>
+                        <span>XX:XX:XX</span>
```

Comment:
Please, employ semantic element `<time>` with `datetime` attribute for better accessibility.

Reply:
Fixed